services/ui-src/src/components/tables/SortableTable.tsx

```diff
@@ -57,6 +61,15 @@ export const SortableTable = ({
     },
   });
 
+  useEffect(() => {
+    const relKeys = Object.keys(headerRefs.current);
+    const refLabels = relKeys.reduce((obj, key) => {
```

Comment:
nit: is this supposed to be `refKeys` or is `relKeys` something?
```suggestion
    const refKeys = Object.keys(headerRefs.current);
    const refLabels = refKeys.reduce((obj, key) => {
```

Reply:
yeah `rel` is a unique identifier, this is code that Bangbay provided to address accessibility failure around button text.